src/Catalyst.Node.POA.CE/Program.cs

```diff
@@ -0,0 +1,201 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Autofac;
+using Autofac.Core;
+using Catalyst.Abstractions;
+using Catalyst.Abstractions.Cli;
+using Catalyst.Abstractions.DAO;
+using Catalyst.Abstractions.IO.Observers;
+using Catalyst.Abstractions.Types;
+using Catalyst.Core.Lib;
+using Catalyst.Core.Lib.Cli;
+using Catalyst.Core.Lib.DAO;
+using Catalyst.Core.Lib.Kernel;
+using Catalyst.Core.Modules.Authentication;
+using Catalyst.Core.Modules.Consensus;
+using Catalyst.Core.Modules.Cryptography.BulletProofs;
+using Catalyst.Core.Modules.Dfs;
+using Catalyst.Core.Modules.Hashing;
+using Catalyst.Core.Modules.KeySigner;
+using Catalyst.Core.Modules.Keystore;
+using Catalyst.Core.Modules.Kvm;
+using Catalyst.Core.Modules.Ledger;
+using Catalyst.Core.Modules.Mempool;
+using Catalyst.Core.Modules.P2P.Discovery.Hastings;
+using Catalyst.Core.Modules.Rpc.Server;
+using Catalyst.Core.Modules.Web3;
+using Catalyst.Modules.POA.Consensus;
+using Catalyst.Modules.POA.P2P;
+using Catalyst.Protocol.Network;
+using CommandLine;
+
+namespace Catalyst.Node.POA.CE
+{
+    internal class Options
+    {
+        [Option("ipfs-password", HelpText = "The password for IPFS.  Defaults to prompting for the password.")]
+        public string IpfsPassword { get; set; }
+
+        [Option("ssl-cert-password", HelpText = "The password for ssl cert.  Defaults to prompting for the password.")]
+        public string SslCertPassword { get; set; }
+
+        [Option("node-password", HelpText = "The password for the node.  Defaults to prompting for the password.")]
+        public string NodePassword { get; set; }
+
+        [Option('o', "overwrite-config", HelpText = "Overwrite the data directory configs.")]
+        public bool OverwriteConfig { get; set; }
+
+        [Option("network-file", HelpText = "The name of the network file")]
+        public string OverrideNetworkFile { get; set; }
+    }
+
+    public static class Program
+    {
+        private static readonly Kernel Kernel;
+
+        static Program()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Initialize all 'static fields' inline and remove the 'static constructor'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4594394)